sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -198,8 +205,24 @@ private void startTracing(final @NotNull Activity activity) {
         appStartSpan =
             transaction.startChild(
                 getAppStartOp(coldStart), getAppStartDesc(coldStart), appStartTime);
+        // The first activity ttidSpan should start at the same time as the app start time
+        ttidSpan = transaction.startChild("TTID", activityName + ".ttid", appStartTime);
```

Comment:
I guess we'd need to raise it with the TSC, because this is can affect performance issues and so on and we have a documented list of span ops https://develop.sentry.dev/sdk/performance/span-operations/. You said you're on PTO next week? I can bring it up instead, if you want me to.

Reply:
yes, please
thanks